### ISSUE TITLE
License

### ISSUE THREAD

**Issue by fakerybakery** [jasonppy/VoiceCraft#1]
Hi,
Thank you for releasing VoiceCraft! It's super cool and I'm really impressed by the quality. Do you have any plans to open source it by switching to an open source license? >(Are the model weights based off of XTTS?)
Thanks!

**Comment by jasonppy** [comment]
Thanks!

Neither the model or the codebase are based off of XTTS. We are discussing the licensing issue right now, might change the license in the future

**Comment by fakerybakery** [comment]
Thank you for considering switching to an open source license!

**Comment by irux** [comment]
Yeah! An open source license would be amazing! Any reason that stop you right now? @jasonppy

**Comment by rebotnix** [comment]
A good idea might be a paid open source license. Open source doesn't necessarily mean free, it's a new way to finance the hard work of these projects. A good example was xtts, which was closed down because there was no clear or good business model, it was the old open source idea, but nobody pays for it. In the end, doesn't it make sense for all of us if the authors can't make it in time or other developers don't get paid? What do you think?

**Comment by truedat101** [comment]
There's no reason a dual license can't be applied, i.e. an open license possibly with some restrictions and a proprietary license available via standard licensing agreements.  But I guess I am curious about the licenses used: https://github.com/jasonppy/VoiceCraft/blob/master/LICENSE-MODEL , and the one for software is a CC license share share alike nc.  This all seems a bit haphazard in the world of licensing software that might have patented IP associated.

**Comment by fakerybakery** [comment]
I think if it were dual-licensed with a license prohibiting commercial use it would not count as open source due to the OSI's OSD

**Comment by fakerybakery** [comment]
Just wanted to say, this is one of the best, if not the best, TTS model I've tried so far. It would be amazing if this could be open-sourced.

**Comment by truedat101** [comment]
> I think if it were dual-licensed with a license prohibiting commercial use it would not count as open source due to the OSI's OSD

Hmm, I need to look at the details on this.  Are you referring to https://opensource.org/osd ?  I don't see any line in there that precludes dual licensing, so as long as the OSS license meets the requirements.  I guess I would disagree, but I am not a lawyer, and would need to see some precedent that says that dual licensing defeats the purpose of the OSD as it is written.   You always have a choice with the OSS if you choose to accept that.  And once granted, the copyright holder can't remove that license on the version of software you have received.

**Comment by fakerybakery** [comment]
> > I think if it were dual-licensed with a license prohibiting commercial use it would not count as open source due to the OSI's OSD
> 
> Hmm, I need to look at the details on this. Are you referring to https://opensource.org/osd ? I don't see any line in there that precludes dual licensing, so as long as the OSS license meets the requirements. I guess I would disagree, but I am not a lawyer, and would need to see some precedent that says that dual licensing defeats the purpose of the OSD as it is written. You always have a choice with the OSS if you choose to accept that. And once granted, the copyright holder can't remove that license on the version of software you have received.

Ah, sorry, I misinterpreted. I thought by dual licensing you meant NC and an additional commercial license

**Comment by mantrakp04** [comment]
+1
